.travis.yml

```diff
@@ -15,6 +16,55 @@ env:
   - TEST_SUITE=brakeman
 matrix:
   fast_finish: true
+  include:
+  - sudo: required
+    group: edge
+    virt: lxd
+    dist: bionic
+    arch: arm64-graviton2
+    rvm: 2.5.8
+    env:
+    - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
+    - RUBY_GC_HEAP_INIT_SLOTS=600000
+    - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+    - PARALLEL_TEST_PROCESSORS=2
+    - TEST_SUITE=vmdb PARALLEL=true
+  - sudo: required
+    group: edge
+    virt: lxd
+    dist: bionic
+    arch: arm64-graviton2
+    rvm: 2.5.8
+    env:
+    - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
+    - RUBY_GC_HEAP_INIT_SLOTS=600000
+    - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+    - PARALLEL_TEST_PROCESSORS=2
+    - TEST_SUITE=brakeman
```

Comment:
I think we can drop the 2.5.8 tests.  We already include those for x86, and really their only purpose is to ensure that we don't introduce 2.6 specific syntax (and so those would be caught by the x86-64 tests)

Actually now that I think about it, I wonder if we can just drop the 2.5 tests entirely and instead just include some sort of `ruby -c` test for all Ruby files? 🤔  @jrafanie WDYT?

Reply:
So, test syntax of 2.5 only, not behavior?  Interesting.  I would think we could leverage `ruby -c` for not yet supported rubies to stay on top of syntax changes.  If we're still supporting 2.5, I'm not sure that checking syntax is good enough though.

Reply:
On master, we should only support the Ruby version (and higher) we are planning to release the next version of ManageIQ with.  The release branches should only specify the branch that they were built with.

Reply:
@jrafanie yeah that's the idea.  I'm just looking for ways to save on timing cycles. 😄

@chessbyte Yeah, that's generally what we do.  We test the Ruby we ship with and then usually 1 less (in case we need to downgrade, and it makes the developer experience a bit smoother).  In fact, I think we are due to bump to Ruby 2.6/2.7 soon.

Reply:
Yeah, agreed. Are we saying we should keep 2.5 for now?

Reply:
I'll open a separate issue for the jump to Ruby 2.7

Reply:
#21008